README.md

```diff
@@ -8,7 +8,10 @@
 This repository contains the Go software development kit for implementing a
 processor for [Conduit](https://github.com/conduitio/conduit).
 
-## Get started
+Note: if you'd like to use another language for writing processors, feel free to
+[open an issue](https://github.com/ConduitIO/conduit/issues) and request a processor SDK for a specific language.
```

Comment:
Maybe we could even link to creating a feature request directly.

```suggestion
[open an issue](https://github.com/ConduitIO/conduit/issues/new?assignees=&labels=feature%2Ctriage&projects=&template=1-feature-request.yml&title=Feature%3A+%3Ctitle%3E) and request a processor SDK for a specific language.
```

Reply:
I'm not sure if I'd mention this in the readme at all, since this is specifically the Go SDK, why concern ourselves with other languages here? 🤷 The docs on conduit.io are more general, I have nothing against it there.